src/main/java/duke/load.java

```
@@ -0,0 +1,131 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class load {
+    /**
+     * Parses an array of tokens from the database,
+     * and creates the corresponding class object,
+     * before adding it to an array of Tasks.
+     *
+     * @param taskList List of tasks to be updated.
+     * @param tokens Array of information of class object to be added.
+     */
+    public static void addTask(List<Task> taskList, String[] tokens){
+        String description, by, start, end;
+        boolean done;
+        Task newTask;
+
+        try {
+            switch (tokens[0]) {
+            case "todo":
+                description = tokens[1];
+                done = tokens[2].contains("X");
+
+                newTask = new ToDo(description);
+                newTask.setDone(done);
+                taskList.add(newTask);
+                break;
+            case "deadline":
+                description = tokens[1];
+                by = tokens[2];
+                done = tokens[3].contains("X");
+
+                newTask = new Deadline(description, by);
+                newTask.setDone(done);
+                taskList.add(newTask);
+                break;
+            case "event":
+                description = tokens[1];
+                start = tokens[2];
+                end = tokens[3];
+                done = tokens[4].contains("X");
+
+                newTask = new Event(description, start, end);
+                newTask.setDone(done);
+                taskList.add(newTask);
+                break;
+            }
+        } catch (MissingParamsException ignored) {
+
```

try not to leave empty catch blocks